test_scripts/Polices/Validation_of_PolicyTables/preloaded_pt_validation_rules_for_required_parameters.lua

```diff
@@ -0,0 +1,113 @@
+---------------------------------------------------------------------------------------------
+-- Description:
+-- Behavior of SDL during start SDL in case when PreloadedPT has no required parameters
+-- 1. Used preconditions:
+-- Delete files and policy table from previous ignition cycle if any
+-- Do not start default SDL
+-- 2. Performed steps:
+-- Create PreloadedPolicyTable file without one required parameter
+-- Start SDL
+
+-- Requirement summary:
+-- [Policies]: PreloadedPolicyTable: Validation rules for required parameters
+--
+-- Expected result:
+-- SDL shuted down
+---------------------------------------------------------------------------------------------
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+local config = require('config')
+
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+local preloaded_pt_file_name = "sdl_preloaded_pt.json"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local json = require("modules/json")
+local SDL = require('modules/SDL')
+
+--[[ Preconditions ]]
+function Test.backup_preloaded_pt()
```

Comment:
Please rewrite name of functions in preconditions to be like: 
function Test:Precondition_BackUp_Preloaded_PT